modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/util/TypeUtils.java

```diff
@@ -226,8 +234,7 @@ private static boolean hasConvertableFields(RelDataType resultType) {
     }
 
     /**
-     * ToInternal.
-     * TODO Documentation https://issues.apache.org/jira/browse/IGNITE-15859
+     * ToInternal. Converts the given value to its presentation used by the execution engine.
```

Comment:
```suggestion
     * Converts the given value to its presentation used by the execution engine.
```

Reply:
Fixed. Removed `TypeUtils::toInternal(val)` as well since it is no longer needed.